apis/elbv2/v1beta1/ingressclassparams_types.go

```diff
@@ -41,6 +41,15 @@ const (
 	LoadBalancerSchemeInternetFacing LoadBalancerScheme = "internet-facing"
 )
 
+// SubnetSelector selects one or more existing subnets.
+type SubnetSelector struct {
+	// ID specifies the resource ID of a subnet. Exactly one of this or `nameContains` must be specified.
+	ID string `json:"id,omitempty"`
+	// NameContains specifies all subnets containing a string in their Name tag. Exactly one of this or `id` must be specified.
+	NameContains string `json:"nameContains,omitempty"`
+	// TODO VPCID string `json:"vpcID,omitempty"`
```

Comment:
Is this the same VPC as the ALB?

Reply:
Good question! A better approach would probably be to add a `vpc` field to `IngressClassParamsSpec` instead of hanging it off of `subnets`.